assets/tests/components/routePicker.test.tsx

```diff
@@ -24,7 +22,13 @@ describe("RoutePicker", () => {
     const tree = renderer
       .create(
         <RoutesProvider routes={routes}>
-          <RoutePicker selectedRouteIds={selectedRouteIds} />
+          <RoutePicker
+            selectedRouteIds={selectedRouteIds}
+            // tslint:disable-next-line: no-empty
+            selectRoute={() => {}}
```

Comment:
**suggestion**: instead of writing the trivial function `() => {}`, how about let's call `jest.fn()`, and then we don't have to keep doing the tslint disable.